Searching_Sorting/ShellSort.py

```diff
@@ -0,0 +1,26 @@
+def shellSort(input_list):

```

Comment:
from PEP8 this should be written in `lower_case_with_underscores`, for e.g.:

`shellSort` -> `shell_sort` or `shellsort`